### DISCUSSION TITLE
建议增加"直接点扩展图标就可翻译"的功能

### DISCUSSION THREAD

**Discussion by suheandzl** [immersive-translate/immersive-translate#552] [Ideas]
### 新功能 | New Feature

1.建议增加"直接点扩展图标就可翻译"的功能
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.扩展弹出页有一定延迟 并且配置后也不太需要下拉菜单选择 建议增加点扩展图标就可翻译的功能 方便实用
2.
3.

**Comment by theowenyoung** [comment]
右键，【翻译网页】 可不可以解决问题呢～

**Reply by suheandzl** [reply]
您好 这个虽然也可以 但我觉得不如直接点图标来的便利 毕竟chrome自带翻译也是点一次就可以翻译的

**Reply by theowenyoung** [reply]
> 您好 这个虽然也可以 但我觉得不如直接点图标来的便利 毕竟chrome自带翻译也是点一次就可以翻译的

其实更方便的是按快捷键. 默认是 `Alt+a`, 可以在界面设置里配置。
